.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Does this file just apply to the entire directory? At some point we'll have an actual spec that it shouldn't apply to. Is there a way to specify that? Sorry, I don't know a lot about prettier.

Reply:
The spec will be a `.bs` file, so that's good to know.